roles/glance/templates/etc/glance/glance-api.conf

```diff
@@ -4,6 +4,8 @@ verbose = {{ glance.logging.verbose }}
 
 {% if glance.store_swift %}
 default_store = swift
+{% elif glance.store_ceph %}
```

Comment:
we should probably use `{% elif glance.store_ceph|bool %}` here, and update the store_swift one as well.   We've seen yml->bool go bad before.